packages/core/utils/src/install-id.ts

```diff
@@ -0,0 +1,14 @@
+import crypto from 'crypto';
+import { machineIdSync } from 'node-machine-id';
+
+export default (projectId: string, installId: string) => {
+  if (installId) return installId;
```

Comment:
I don't get why we have the installId in parameters if we return it if it's not a nullable. Shouldn't the function be not invoked if the caller already has the information?

Reply:
Wouldn't it be less obfuscated to do `installId ?? getInstallId(projectId)` where you need the installId? 

Another one: Could you do a named export? using `export default` reduce the discoverability of the code 👍

Reply:
What do you think @innerdvations?

Reply:
Yes, I can make a named export!